__tests__/__mocks__/documentData.js

```diff
@@ -10,14 +10,15 @@ const developerData = {
   password: 'password',
 };
 
+const developerData2 = {
```

Comment:
it's not great practice to include numbers in variable names - please change this variable name